barretenberg/cpp/src/barretenberg/translator_vm/translator_proving_key.hpp

```diff
@@ -38,10 +39,13 @@ class TranslatorProvingKey {
     {
         PROFILE_THIS_NAME("TranslatorProvingKey(TranslatorCircuit&)");
 
+        // WORKTODO: the methods below just set the constant values TRANSLATOR_VM_FIXED_SIZE and
+        // TRANSLATOR_VM_FIXED_SIZE * INTERLEAVING_GROUP_SIZE
         compute_mini_circuit_dyadic_size(circuit);
         compute_dyadic_circuit_size();
-        proving_key = std::make_shared<ProvingKey>(dyadic_circuit_size, std::move(commitment_key));
```

Comment:
This line was redundantly constructing `ProvingPolynomials` via the `ProvingKey` constructor. It was doing so with input `dyadic_circuit_size` instead of the intended `mini_circuit_size`, presumably consuming much more memory than necessary